scripts/config/org/icepear/echarts/origin/chart/treemap/treemap-series-option.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "TreemapSeriesOption",
+    "type": "interface",
+    "extends": [
+        "SeriesOption",
+        "TreemapStateOption",
+        "BoxLayoutOptionMixin",
+        "RoamOptionMixin",
+        "TreemapSeriesVisualOption"
+    ],
+    "fields": [
+        { "name": "type", "types": ["String"], "default": "treemap"},
+        { "name": "size", "types": ["Number[]", "String[]"]},
+        { "name": "sort", "types": ["Boolean"], "default": true},
```

Comment:
Why does it default to `true`? Let's not set default value for any field except type. We don't generate option with fields that users don't set explicitly.

Reply:
There is a comment saying the default is 'desc' which means 'true' equivaltly,  in the source codes of ECharts.
 ``` /**
     * If sort in desc order.
     * Default to be desc. asc has strange effect
     */
    sort?: boolean | 'asc' | 'desc'
```
I have no preference for this though, can set it to ["Boolean", "String"].